src/en/guide/how-to/orm.md

```diff
@@ -2,32 +2,32 @@
 
 >  How do I use SQLAlchemy with Sanic ?
 
-All ORM tools can work with Sanic, but non-async ORM tool have a impact on Sanic performance. 
-There are some orm packages who support 
+All ORM tools can work with Sanic, but non-async ORM tool have a impact on Sanic performance.
+There are some orm packages who support
 
-At present, there are many ORMs that support asynchrony, among which the better ones are：
+At present, there are many ORMs that support asynchronicity. Two of the more common libraries are：
 
-[SQLAlchemy 1.4](https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html)  [tortoise-orm](https://github.com/tortoise/tortoise-orm)
-
-Don't know how to integrate it into sanic ?  don't worry, there is a simple usage:
+- [SQLAlchemy 1.4](https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html)
+- [tortoise-orm](https://github.com/tortoise/tortoise-orm)
 
+Integration in to your Sanic application is fairly simple:
 
 ## SQLAlchemy
 
-Yeah, as you see, [SQLAlchemy 1.4](https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html) has native support for asyncio, Sanic finally can play happily with sqlalchemy.
+Because [SQLAlchemy 1.4](https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html) has addednative support for asyncio, Sanic can finally work well with sqlalchemy. Be aware that this functionality is still considered *beta* by the SQLAlchemy project.
```

Comment:
```suggestion
Because [SQLAlchemy 1.4](https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html) has added native support for `asyncio`, Sanic can finally work well with SQLAlchemy. Be aware that this functionality is still considered *beta* by the SQLAlchemy project.
```